benchmark_runner/common/ocp_resources/odf/template/01_sgdisks_template.sh

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env sh
 
-sgdisk_list=$1
-oc get nodes -l node-role.kubernetes.io/worker= -o jsonpath="{range .items[*]}{.metadata.name}{'\n'}{end}" |  xargs -I{} oc debug node/{} -- chroot /host sh -c "$sgdisk_list"
+node0=$1
+node0_sgdisk=$2
+node1=$3
+node1_sgdisk=$4
+node2=$5
+node2_sgdisk=$6
+
+oc debug node/$node0 -- chroot /host sh -c "$node0_sgdisk"
+oc debug node/$node1 -- chroot /host sh -c "$node1_sgdisk"
+oc debug node/$node2 -- chroot /host sh -c "$node2_sgdisk"
```

Comment:
Again, can you do this without assuming exactly three nodes?  Something like this:

```
while (($# >= 2)) ; do
    node=$1
    sgdisk_command=$2
    oc debug "node/$node"  -- chroot /host sh -c "$sgdisk_command"
    shift 2
done